apps/web/src/utils/profanity.ts

```diff
@@ -0,0 +1,9 @@
+import Filter from 'bad-words';
+import overrideWords from './bad-word-overrides.json';
+
+
+export function containsProfanity(str: string): boolean {
+    const f = new Filter();
```

Comment:
can move this outside the function otherwise it constructs a new class every invocation

Reply:
oh shoot, yeah I'll do that